control/statefbk.py

```diff
@@ -684,7 +685,7 @@ def create_statefbk_iosystem(
         hull of the scheduling points, the gain at the nearest point is
         used.
 
-    type : 'linear' or 'nonlinear', optional
+    controller_type : 'linear' or 'nonlinear', optional
```

Comment:
That's an API change. The milestone should be 0.10.0 (or 1.0.0) then,

Reply:
I put in code to handle legacy calls, so `type` will still work.  See line 732.